docs/source/distributed.rst

```diff
@@ -55,14 +56,15 @@ Currently three initialization methods are supported:
 TCP initialization
 ^^^^^^^^^^^^^^^^^^
 
-Initialization will utilize a network address reachable from all processes.
-If the address belongs to one of the machines, initialization requires that all processes
-have manually specified ranks.
+There are two ways to intialize using TCP, both requiring a network address
+reachable from all processes and a desired ``world_size``. The first way
+requires specifying an address that belongs to the rank 0 process. This form of
+initialization requires that all processes have manually specified ranks.
```

Comment:
It seems a bit misleading now, because you only have to manually assign ranks if you use the rank 0 addr method (multicast can assign them automatically). Perhaps split this paragraph at "The first way" to indicate that it doesn't apply to both ways?